drivers/nrf24l01p/Kconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023 Inria
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig MODULE_NRF24L01P
+    bool
+    prompt "nrf24l01p radio" if !(MODULE_NETDEV_DEFAULT && HAVE_NRF24L01P)
+    default (MODULE_NETDEV_DEFAULT && HAVE_NRF24L01P)
+    depends on TEST_KCONFIG
+    select MODULE_PERIPH_GPIO
+    select MODULE_PERIPH_GPIO_IRQ
+    select MODULE_PERIPH_SPI
+    select MODULE_ZTIMER
```

Comment:
I think it is better to just select `ZTIMER_USEC` here.  This is needed due to some circular dependencies in the modeling.

Reply:
Thanks, I applied that change and it's still building fine and generated binary is the same as without kconfig.